lib/modules/repository.js

```diff
@@ -161,21 +161,23 @@ class Repository {
     branchName,
     originalSha,
     convention,
+    commitType,
   }) {
     if (originalSha === undefined) {
-      await this.createFile({ filePath, content, branchName, convention });
+      await this.createFile({ filePath, content, branchName, convention, commitType });
     } else {
       await this.updateFile({
         filePath,
         content,
         branchName,
         originalSha,
         convention,
+        commitType,
       });
     }
   }
 
-  async createOrUpdateFiles({ filesByPath, branchName, convention }) {
+  async createOrUpdateFiles({ filesByPath, branchName, convention, commitType }) {
```

Comment:
On that note, we're handing down a lot of config like variables to these class methods. Perhaps in a later PR one could refactor this so that we set up the branchName, convention, commitType etc. when the Repository class is instantiated.